inference_segmentation.py

```diff
@@ -157,32 +156,24 @@ def segmentation(param,
     Returns:
 
     """
-    subdiv = 2
+    use_hanning = True
```

Comment:
Could this be softcoded? Even if we intend to always use hanning, it seems like it would be nice to set this value from config (default=True) at least to hint the user than some smoothening is going on.

Reply:
Yes, agreed!